Microsoft.Identity.Web/REAME.md

```diff
@@ -0,0 +1,22 @@
+# Microsoft Identity Web
```

Comment:
you might want to resync this one.

Is it an option for you to use Git submodules?

Reply:
Using `$ git clone --recurse-submodules` may not work for users who may not have Git set up in their local dev. environment or those who may want to download the project as zip.